spec/paperclip/storage/fog_spec.rb

```diff
@@ -1,5 +1,6 @@
 require 'spec_helper'
-require 'fog'
+require 'fog/aws'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.